mandatory/2-logic-error.js

```diff
@@ -25,11 +25,11 @@ To run the tests for just this one file, type `npm test -- --testPathPattern 2-l
 */
 
 test("trimWord trims leading and trailing whitespace", () => {
-  expect(trimWord("   CodeYourFuture ")).toEqual("CodeYourFuture");
+  expect(trimWord("CodeYourFuture")).toEqual("CodeYourFuture");
 });
 
 test("trimWord doesn't remove whitespace in the middle of the string", () => {
-  expect(trimWord(" CodeYourFuture teaches coding     ")).toEqual(
+  expect(trimWord("CodeYourFuture teaches coding")).toEqual(
```

Comment:
The good news is your `trimWord` function looks perfect :D However, please **never** change the tests in these files. They are there to indicate if your code is working as it should, and by changing them you are opening yourself to a world of issues. If you think there _is_ a problem with one of the tests, please let myself (or another volunteer) know so we can investigate, but please **don't change them yourself**.